ci/ci.py

```diff
@@ -146,7 +146,7 @@ def install(args):
             setup_keychain(args)
 
 
-def build_engine(platform, with_valgrind = False, with_asan = False, with_vanilla_lua = False, skip_tests = False, skip_codesign = True, skip_docs = False, skip_builtins = False, archive = False):
+def build_engine(platform, with_valgrind = False, with_asan = False, with_vanilla_lua = False, skip_tests = False, skip_codesign = True, skip_docs = False, skip_builtins = False, archive = False, channel = None):
```

Comment:
Always specify a channel. build.py will default to "stable" if no channel is provided.